config/routes.rb

```diff
@@ -8,5 +8,9 @@
     end
   end
 
+  get '/api/users/:id/posts', to: 'api#posts_index'
+  get '/api/users/:id/posts/:id', to: 'api#list_comments'
+  post '/api/users/:id/posts/:id', to: 'api#create_comment'
```

Comment:
- [ ] Hi, as part of the requirements for this project. you need to add two other routes.
- [ ] a Login API endpoint - POST : `/login` or `/login`
- [ ] a Registration API endpoints - POST : /register or /signup

Reply:
Thank you for your feedback @MrBrN197, we have added a login and register endpoint now.